README.rst

```diff
@@ -13,8 +13,8 @@ We'd love for you to `contribute to Girder <CONTRIBUTING.md>`_.
 
 .. |logo| image:: clients/web/static/img/Girder_Favicon.png
 
-.. |build-status| image:: https://travis-ci.org/girder/girder.svg?branch=master
-    :target: https://travis-ci.org/girder/girder
+.. |build-status| image:: https://circleci.com/gh/girder/girder.png?style=shield
```

Comment:
It's not a great omen that this badge shows the default branch (master) is failing.